app/controllers/api/v3/statements_controller.rb

```diff
@@ -12,13 +12,22 @@ def show
     private
 
       def statements_query
-        ::Statements::Query.new(
-          lead_provider: current_lead_provider,
-        )
+        Statements::Query.new
+                         .belonging_to(lead_provider: current_lead_provider)
+                         .by_cohorts(cohorts)
+                         .since(updated_since)
       end
 
       def statement_params
-        params.permit(:id)
+        params.permit(:id, :cohort, filter: %i[cohort updated_since])
+      end
+
+      def cohorts
+        statement_params.dig(:filter, :cohort)&.split(",")
```

Comment:
maybe we can pass the filter as is and attempt the `&.split(",")` in the service to avoid having to test this at the request spec level but we can at the query service level?